control-plane-operator/controllers/hostedcontrolplane/pki/kas.go

```diff
@@ -70,6 +70,10 @@ func ReconcileIngressOperatorClientCertSecret(secret, ca *corev1.Secret, ownerRe
 	return reconcileSignedCert(secret, ca, ownerRef, "system:serviceaccount:openshift-ingress-operator:ingress-operator", []string{"system:serviceaccounts"}, X509UsageClientServerAuth)
 }
 
+func ReconcileKASMetricsClientCertSecret(secret, ca *corev1.Secret, ownerRef config.OwnerRef) error {
+	return reconcileSignedCert(secret, ca, ownerRef, "kas-metrics-client", []string{"kubernetes"}, X509UsageClientServerAuth)
```

Comment:
I would add a prefix to the subject like `system:kas-metrics-client` to avoid any potential clashes with idp names.